tests/ha/fencing.pm

```diff
@@ -25,8 +25,14 @@ sub run {
     check_cluster_state;
     barrier_wait("CHECK_BEFORE_FENCING_END_$cluster_name");
 
-    # Fence the master node
-    assert_script_run 'crm -F node fence ' . get_node_to_join if is_node(2);
+    # Fence the master node with sysrq or crm node fence
+    # Sysrq fencing is more a real crash simulation
+    if ((get_var('USE_SYSRQ_FENCING')) && (get_var('HA_CLUSTER_INIT'))) {
```

Comment:
I think if you move the `get_var('HA_CLUSTER_INIT')` from the if condition into the block itself (on the `script_run()` call), it will accomplish what I am commenting on below.

Reply:
Thanks, I'm going to test it.